### PR TITLE
Update extensibility-node8.md

### PR DESCRIPTION
**Pull request by tamigoodall**
Added new dates
Clarified automatic migration
Moved Migration Assistant up

### CODE REVIEW

**Comment by tamigoodall** [comment]
https://auth0team.atlassian.net/projects/DR/queues/custom/100/DR-159

File: articles/migrations/guides/extensibility-node8.md [hunk 1]
```diff
@@ -20,11 +20,22 @@ useCase:
 ---
 # Migration Guide: Extensibility and Node 8
 
-Beginning April 30, 2018, [Node.js v4 will be going out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team will no longer be back-porting critical security fixes to this version and this _could_ expose your extensibility code to security vulnerabilities.
```
Comment (charlesjohnson) [inline articles/migrations/guides/extensibility-node8.md]: This date stays 2018.

Comment (tamigoodall) [reply articles/migrations/guides/extensibility-node8.md]: Fixed.

**Comment by tamigoodall** [comment]
Heroku link: https://docs-content-staging-pr-7547.herokuapp.com/docs/migrations/guides/extensibility-node8

**Comment by charlesjohnson** [comment]
I tried to push to this branch, and I don't have perms (understandably). What's the best way to move forward?

**Review by khorne3 — APPROVED** [review]

**Comment by charlesjohnson** [comment]
Moved my updates I couldn't push here to #7559

**Pull request merged**